File: cpp/kalkulator.cpp
/*
 * kalkulator.cpp
 */
 


#include <iostream>
#include <cmath>
using namespace std;

int main(int argc, char **argv)
{  char znak; 
    int a , b ;
    
    cout << "Podaj znak " << endl;
    cin >> znak ;
    
    cout << "Podaj dwie liczby " ;
    cin >> a >> b;

     // + ,- , *, /
    if (znak == '+')
    { cout << "Suma" << a+ b ;
    }
        else  if (znak == '-')
    { cout << "Różnica" << a - b ;
    }
        else if (znak == '/')
    { cout << "Iloraz" << a /b ;
    }
        else if (znak == '*')
    { cout << "Iloczyn" << a * b ;
    }
        
        


    return 0;
}

File: powtórzenie klasa 2/kalkulator.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  bez nazwy.py


def suma(a, b):
    wynik = a+b
    return wynik
    
    
def roznica(a, b):
    wynik = a-b
    return wynik
    
    
def iloczyn(a, b):
    wynik = a*b
    return wynik
    
    
def iloraz(a, b):
    wynik = a/b
    return wynik



def main(args):
    a = int(input("Podaj 1. liczbę: "))
    print (a)
    b = int(input("Podaj 2. liczbę: "))
    print (b)
    
    print("Suma: ", suma(a, b))
    print("Różnica:",roznica(a, b ))
    print("Iloczyn: ",iloczyn(a,b))
    print("Iloraz: ", iloraz(a, b))
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
